projects/1.18/assets/nock-enough-arrows/nock_enough_arrows/lang/zh_cn.json

```
@@ -0,0 +1,98 @@
+  "entity.nock_enough_arrows.torch_arrow": "火把箭",
+  "entity.nock_enough_arrows.soul_torch_arrow": "灵魂火把箭",
+  "entity.nock_enough_arrows.redstone_torch_arrow": "红石火把箭",
+  "entity.nock_enough_arrows.ethereal_arrow": "虚幻箭",
+  "entity.nock_enough_arrows.dousing_arrow": "浇灌箭",
+  "entity.nock_enough_arrows.blossom_arrow": "绽放箭",
+  "entity.nock_enough_arrows.growing_arrow": "成长箭",
+  "entity.nock_enough_arrows.drill_arrow": "钻头箭",
+  "entity.nock_enough_arrows.split_arrow": "分裂箭",
+  "entity.nock_enough_arrows.party_arrow": "派对箭",
+  "entity.nock_enough_arrows.repulsive_arrow": "厌恶箭",
+  "entity.nock_enough_arrows.echoing_arrow": "回响箭",
+
+  "enchantment.nock_enough_arrows.stockpile": "存储",
+  "enchantment.nock_enough_arrows.recovery": "回收",
+
+  "advancements.fletching.root.title": "足够多箭矢",
+  "advancements.fletching.root.description": "稳住手……",
+  "advancements.fletching.get_quiver.title": "恐惧中颤抖吧!",
+  "advancements.fletching.get_quiver.description": "制作一个箭袋来储存箭失",
+  "advancements.fletching.pufferfish_arrow.title": "更大的鱼在飞",
+  "advancements.fletching.pufferfish_arrow.description": "河豚箭的制作工艺",
+  "advancements.fletching.explosive_arrow.title": "谨慎风向",
+  "advancements.fletching.explosive_arrow.description": "为箭矢装上爆炸箭头",
+  "advancements.fletching.slime_arrow.title": "三角函数!",
+  "advancements.fletching.slime_arrow.description": "为箭矢装上史莱姆箭头",
+  "advancements.fletching.prismarine_arrow.title": "逆流而上",
+  "advancements.fletching.prismarine_arrow.description": "为箭矢装上海晶石箭头",
+  "advancements.fletching.hookshot_arrow.title": "旧西部",
+  "advancements.fletching.hookshot_arrow.description": "为箭矢装上钩子箭头",
+  "advancements.fletching.message_arrow.title": "来自另一个世界的问候",
+  "advancements.fletching.message_arrow.description": "为箭矢装上信息箭头",
+  "advancements.fletching.teleportation_arrow.title": "就像末影人一样!",
+  "advancements.fletching.teleportation_arrow.description": "为箭矢装上传送箭头",
+  "advancements.fletching.ink_arrow.title": "鱿鱼的复仇",
+  "advancements.fletching.ink_arrow.description": "为箭矢装上墨水箭头",
+  "advancements.fletching.torch_arrow.title": "燃烧吧!",
+  "advancements.fletching.torch_arrow.description": "为箭矢装上火把箭头",
+  "advancements.fletching.soul_torch_arrow.title": "照亮灵魂",
+  "advancements.fletching.soul_torch_arrow.description": "为箭矢装上灵魂火把箭头",
+  "advancements.fletching.redstone_torch_arrow.title": "遥控信号",
+  "advancements.fletching.redstone_torch_arrow.description": "为箭矢装上红石火把箭头",
+  "advancements.fletching.ethereal_arrow.title": "进入星界位面",
+  "advancements.fletching.ethereal_arrow.description": "为箭矢装上虚幻箭头",
+  "advancements.fletching.dousing_arrow.title": "呼叫消防局！！！",
+  "advancements.fletching.dousing_arrow.description": "为箭矢装上浇灌箭头",
+  "advancements.fletching.blossom_arrow.title": "邻居家的花园",
+  "advancements.fletching.blossom_arrow.description": "为箭矢装上绽放箭头",
+  "advancements.fletching.growing_arrow.title": "向hank 询问此事",
+  "advancements.fletching.growing_arrow.description": "为箭矢装上成长箭头",
+  "advancements.fletching.drill_arrow.title": "我们做了一个小的钻探",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |

怪

```suggestion
  "advancements.fletching.drill_arrow.title": "我们做了一个小的钻探",
```
原文：**We Do a little Drilling**